java/conf/cobbler/snippets/certificate-fetch

```diff
@@ -0,0 +1,10 @@
+<!-- Fetch certificate from server to be able to validate https repositories -->
+<script>
+    <filename>certificate-fetch.sh</filename>
+    <source>
+        <![CDATA[
+        curl --insecure -o /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT  http://$redhat_management_server/pub/RHN-ORG-TRUSTED-SSL-CERT
```

Comment:
Using --insecure with a **http** url does not make sense. Should we directly call **https** instead?

Reply:
I added is as a precaution. URLs in autoyast template uses http as well but, at least in case of this user, they were automatically elevated to https.

Reply:
I think we should change all URLs to https directly. Reason: I know about some customers who block Port 80.
In this case they would not even reach the code which send the redirect.